scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  dob DATE,
+  speciality CHARACTER VARYING
+)
+
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  apptdate TEXT,
+  appttime TEXT,
+  comments TEXT,
+  patient_id INTEGER REFERENCES appointments(id),
```

Comment:
Also: Unfortunately, because of the order of these `CREATE TABLE` commands, this will not work - the reason is that `patient_id INTEGER REFERENCES patients(id)` would, as written, refer to a table (`patients`) that hasn't been created yet.